src/views/events.js

```diff
@@ -141,6 +141,16 @@ var EventsView = JuttleView.extend({
         this.target.stream_end();
     },
 
+    _validateTimeField : function(time) {
+        // if in timechart overlay, timeField needs to actually be a Date
+        if (this._targetView) {
+            return _.isDate(time) && !_.isNaN(time.getTime());
```

Comment:
molasses has been known to travel at speeds of up to [35 mph](https://en.wikipedia.org/wiki/Great_Molasses_Flood), but point taken, thx for the heads up.